src/index.ts

```diff
@@ -150,3 +150,5 @@ export class Ollama {
 		return json.embedding;
 	}
 }
+
+export default new Ollama();
```

Comment:
This will throw an error in environments where `fetch` cannot be automatically obtained.

Reply:
Ah shoot, didn't notice this. I'll see if I can find some way to make this safe.